test-support/helpers/sl/synchronous/requires.js

```diff
@@ -3,66 +3,66 @@ import Ember from 'ember';
 /**
  * Test that an argument passed to a function is of the required type(s).
  *
- * @param  {function} methodUnderTest
- * @param  {array}    requiredTypes
- * @return {object}
+ * @function
+ * @param {Function} methodUnderTest
+ * @param {Array} requiredTypes
+ * @returns {Object}
  */
 export default function( methodUnderTest, requiredTypes ) {
-    var typesToTest = {
-            'number' : {
-                required  : false,
-                testValue : 123987465,
-                message   : 'Parameter was a number'
-            },
-            'string' : {
-                required  : false,
-                testValue : 'testString',
-                message   : 'Parameter was a string'
-            },
-            'array' : {
-                required  : false,
-                testValue : [],
-                message   : 'Parameter was an array'
-            },
-            'object' : {
-                required  : false,
-                testValue : {},
-                message   : 'Parameter was an object'
-            },
-            'function' : {
-                required  : false,
-                testValue : function(){},
-                message   : 'Parameter was a function'
-            },
-            'undefined' : {
-                required  : false,
-                testValue : undefined,
-                message   : 'Parameter was undefined'
-            },
-            'boolean' : {
-                required  : false,
-                testValue : true,
-                message   : 'Parameter was a boolean'
-            }
+    let typesToTest = {
+        'number': {
+            required: false,
+            testValue: 123987465,
+            message: 'Parameter was a number'
+        },
+        'string': {
+            required: false,
+            testValue: 'testString',
+            message: 'Parameter was a string'
+        },
+        'array': {
+            required: false,
+            testValue: [],
+            message: 'Parameter was an array'
+        },
+        'object': {
+            required: false,
+            testValue: {},
+            message: 'Parameter was an object'
+        },
+        'function': {
+            required: false,
+            testValue() {},
+            message: 'Parameter was a function'
         },
-        testsThatHaveFailed = [],
-        assertionThrown,
-        assertionPassed,
-        property,
-        parameter;
+        'undefined': {
+            required: false,
+            testValue: undefined,
+            message: 'Parameter was undefined'
+        },
+        'boolean': {
+            required: false,
+            testValue: true,
+            message: 'Parameter was a boolean'
+        }
+    };
+    let testsThatHaveFailed = [];
+    let assertionThrown;
+    let assertionPassed;
+    let property;
+    let parameter;
 
```

Comment:
Move these let statements to a location closets to where they are being used, respective of the scope they require.